packages/ui/src/components/va-data-table/hooks/useFilterable.ts

```diff
@@ -0,0 +1,31 @@
+import { Ref, computed, watch } from 'vue'
+import { TableRow } from './useRows'
+
+export type TFilteringFn = (source: any) => boolean
+
+// TODO: `emit` must not be any!
```

Comment:
@RVitaly1978 please, take a look at what we can do with `emit: any` here and in other hooks. There must be a standard Vue declaration for emits, we should use it.

Reply:
Done.